lib/views/LineGraph.js

```diff
@@ -13,6 +13,7 @@ class LineGraph extends Graph {
         this.guideClassName = 'y_guide-' + this.id;
         this.guideColor = options.guideColor || this.color;
         this.interpolate = options.interpolate || 'linear';
+        this.shouldAnimate = options.shouldAnimate || false;
```

Comment:
I'd consider moving the `this.shouldAnimate = ...` lines into the base class constructor so they don't need to be duplicated across all `Graphs`: https://github.com/nutonian/nugget/blob/13789d616add4bf14f29dc647bbaef26e0ee18d9/lib/views/Graph.js#L7